### ISSUE TITLE
Test for Technique G60: Playing a sound that turns off automatically within three seconds

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#246]
http://www.w3.org/TR/WCAG20-TECHS/G60.html

The purpose of this technique is to allow authors to play a sound on their Web page but avoid the problem of users not being able to use their screen readers due to interference by the content sound. It also allows the author to avoid putting controls on the Web page to control the sound - and the problem faced by users with screen readers in finding the control (when unable to hear their screen reader).

The technique is simple. The sound plays for 3 or less seconds and stops automatically.
## Procedure
- Load the Web page
- Check that all sound that plays automatically stops in 3 seconds or less
## QUAIL

HTML5: check mediaelement status muted and paused after currentTime > 3
